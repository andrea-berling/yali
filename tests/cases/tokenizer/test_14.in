5720.9373
