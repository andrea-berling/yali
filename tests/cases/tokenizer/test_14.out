NUMBER 5720.9373 5720.9373
EOF  null
